src/views/shared/SecurityBeacon.js

```diff
@@ -133,6 +133,7 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
       this.listenTo(options.appState, 'loading', function (isLoading) {
         this.$el.toggleClass('beacon-loading', isLoading);
       });
+      this.options.appState.set('beaconType', 'security');
```

Comment:
More as a note to myself: I just noticed that the Beacons don't share a common view class - we should fix that in the future.